README.md

```diff
@@ -141,75 +143,56 @@ print(
 ```
 
 You can also run an instrumented binary directly from the REPL:
+
 ```python
 $ polytracker
 PolyTracker (4.0.0)
 https://github.com/trailofbits/polytracker
 Type "help" or "commands"
 >>> trace = run_trace("path_to_binary", "path_to_input_file")
 ```
+
 This will automatically run the instrumented binary in a Docker container, if necessary.
 
 > :warning: **If running PolyTracker inside Docker or a VM**: PolyTracker can be very slow if running in a virtualized
 > environment and either the input file or, especially, the output database are located in a directory mapped or mounted
 > from the host OS. This is particularly true when running PolyTracker in Docker from a macOS host. The solution is to
 > write the database to a path inside of the container/VM and then copy it out to the host system at the very end.
 
-The optional `POLYTRACE` environment variable can be set to `POLYTRACE=1` to produce a basic-block
-level trace of the program.
-
 The Python API documentation is available [here](https://trailofbits.github.io/polytracker/latest/).
 
 ## Runtime Parameters and Instrumentation Tuning
 
-At runtime, PolyTracker instrumentation looks for a number of configuration parameters either specified through
-environment variables or a local configuration file. This allows one to modify instrumentation parameters without
-needing to recompile the binary.
+At runtime, PolyTracker instrumentation looks for a number of configuration parameters specified through
+environment variables. This allows one to modify instrumentation parameters without needing to recompile the binary.
 
 ### Environment Variables
 
 PolyTracker accepts configuration parameters in the form of environment variables to avoid recompiling target programs.
 The current environment variables PolyTracker supports is:
 
 ```
-POLYPATH: The path to the file to mark as tainted 
+<!-- POLYPATH: The path to the file to mark as tainted  -->
 
-POLYTTL: This value is an initial "strength" value for taint nodes, when new nodes are formed, the average is taken. When the TTL value is 0, the node is considered clean. 
+<!-- POLYTTL: This value is an initial "strength" value for taint nodes, when new nodes are formed, the average is taken. When the TTL value is 0, the node is considered clean.  -->
 
-POLYSTART: Start offset to track 
+<!-- POLYSTART: Start offset to track  -->
 
-POLYEND: End offset to track
+<!-- POLYEND: End offset to track -->
```

Comment:
These are still visible in the Github markdown viewer: [link](https://github.com/trailofbits/polytracker/tree/6460-remove-information-about-polypath-in-readmemd#environment-variables)
I would also remove them completely

Reply:
Hopefully resolved completely now.